scipy/stats/_continuous_distns.py

```diff
@@ -3528,6 +3530,47 @@ def _ppf(self, q, a, b):
 
 johnsonsu = johnsonsu_gen(name='johnsonsu')
 
+class johnsonsl_gen(rv_continuous):
+    r"""A Johnson SL continuous random variable.
+    %(before_notes)s
+
+    See Also
+    --------
+    johnsonsb
+    johnsonsu
+
+    Notes
+    -----
+    The probability density funcition for a `johnsonsl` is::
```

Comment:
please align the docstring to the other distributions: use mathmode, specify range of a and b